workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
```

Comment:
Is `4 ` a number chosen for any particular reason here, or is [it random](https://www.xkcd.com/221/)